bouncer/tests/all_swaps.ts

```diff
@@ -68,5 +68,14 @@ async function main() {
       });
   });
 
+  // Not doing BTC due to encoding complexity in vault_swap. Will be fixed once SDK supports it.
```

Comment:
we have a ticket for this?

Reply:
This should be part of adding the logic in the SDK and broker api to create Bitcoin Vault swaps and other vault swaps for that matter. It's encapsulated into PRO-1642 but not sure if there is some ticket on the product side to expand vault swaps in the SDK. There should be or at least they must have planned for it.